modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/BaseExportImportTestCase.java

```diff
@@ -47,6 +49,114 @@
  */
 public abstract class BaseExportImportTestCase {
 
+	public void testExportImport(
+			JSONObject actualJSONObject, JSONObject expectedJSONObject,
```

Comment:
Since every time we are using the actualJsonObject and expectedJsonObject we are changing them to strings before using them properly, maybe its better if this method already receives String objects.

Reply:
Also we will probably have to change the name to `actualJSONString` and `expectedJSONString` ? what do you think?